### DISCUSSION TITLE
Automation

### DISCUSSION THREAD

**Discussion by ippocratis** [hardcore-sushi/DroidFS#190] [Ideas]
Auto import/encrypt and then delete(?) specific folders on a schedule

Usecase:
periodic/sheduled  exports of e.g. 2fa vaults (aegis) , SMS/call logs,  password vaults etc from their apps can be auto imported/encrypted in to droidfs

**Comment by hardcore-sushi** [comment]
I don't think we can export data from other apps without root access. Moreover, Android makes hard to run a background service on a specific schedule.
